src/components/Map/components/CardSearchDetails.tsx

```diff
@@ -39,26 +80,59 @@ const CardSearchDetails = ({ result, onClick, onClickClose }: SearchResult) => {
       className={isEligible ? 'eligible' : 'ineligible'}
       isClickable
     >
-      <header onClick={onClickHandler}>{result.address}</header>
+      <header onClick={onClickHandler}>{storedAddress.address}</header>
       {onClickClose && (
-        <button className="closeButton" onClick={onCloseHandler} />
+        <button
+          type="button"
+          title="Fermer"
+          className="closeButton"
+          onClick={onCloseHandler}
+        />
       )}
       <section>
         <EligibilityResult isEligible={isEligible}>
           {isEligible
             ? `Bonne nouvelle ! Un réseau de chaleur passe à proximité de cette adresse.`
             : `D'après nos données, il n'y a pour le moment pas de réseau de chaleur à proximité de cette adresse.`}
+          <div>
+            <strong>
+              {isEligible &&
+                !isNaN(parseFloat(distance)) &&
+                `Le réseau passe à ${readableDistance}`}
+            </strong>
+          </div>
         </EligibilityResult>
-        <div>
-          {!isNaN(parseFloat(distance)) &&
-            `Distance au réseau ${
-              distance < 1
-                ? '< 1m'
-                : distance >= 1000
-                ? `: ${distance / 1000}km`
-                : `: ${distance}m`
-            }`}
-        </div>
+        {!contactFormVisible && storedAddress.contacted ? (
+          <MessageConfirmBox>
+            <Icon
+              name="fr-icon-success-fill"
+              size="lg"
+              color="#78EB7B"
+              iconPosition="right"
+            />
+            Message dejà envoyé
```

Comment:
```suggestion
            Demande envoyée
```

Ou Demande pirse en compte